src/xtgeo/grid3d/grid.py

```diff
@@ -2403,19 +2555,19 @@ def refine_vertically(self, rfactor, zoneprop=None):
 
     def report_zone_mismatch(
         self,
-        well=None,
-        zonelogname="ZONELOG",
-        zoneprop=None,
-        onelayergrid=None,
-        zonelogrange=(0, 9999),
-        zonelogshift=0,
-        depthrange=None,
-        perflogname=None,
-        perflogrange=(1, 9999),
-        filterlogname=None,
-        filterlogrange=(1e-32, 9999.0),
-        resultformat=1,
-    ):
+        well: Optional[xtgeo.Well] = None,
+        zonelogname: str = "ZONELOG",
```

Comment:
Another case where I think preferring just `Well` is a bit more pleasant, but just my opinion

Reply:
We could, but then we risk that the Well object becomes a global. So we i think we need to live with the Optianal for now.

Also, again, super-afraid of breaking user space, is there anyone out there that has built logic ontop of this beeing None?